CameraOverlay/commons.py

```diff
@@ -31,15 +31,14 @@ def set_overlay(new_overlays, directory=CURRENT_DIRECTORY):
 def read_configs(directory=CURRENT_DIRECTORY):
 	filepath = os.path.join(directory, CONFIG_FILE)
 	if not os.path.isfile(filepath):
-		return {}
+		configs = {}
 	else:
 		with open(filepath) as f:
 			configs = json.load(f)
-
-			current_device = os.getenv('MHP_CAMERA')
-			configs['device'] = current_device
-			configs['overlays'] = get_overlays()
-			return configs
+	current_device = os.getenv('MHP_CAMERA')
+	configs['device'] = current_device
+	configs['overlays'] = get_overlays()
```

Comment:
If `configs.json` either isn't present or doesn't define `activeOverlay`, there needs to be a default overlay. (could just pick either for now)